src/cleo/application.py

```diff
@@ -170,11 +164,6 @@ def catch_exceptions(self, catch_exceptions: bool = True) -> None:
     def is_single_command(self) -> bool:
         return self._single_command
 
-    def set_solution_provider_repository(
```

Comment:
did you have a replacement in mind for this? `poetry` [uses it](https://github.com/python-poetry/poetry/blob/8dbe41e507d82c15c72105b0b14b8c18035dc772/src/poetry/console/application.py#L180) to suggest a [solution](https://github.com/python-poetry/poetry/blob/8dbe41e507d82c15c72105b0b14b8c18035dc772/src/poetry/mixology/solutions/solutions/python_requirement_solution.py) for conflicting python requirements

Reply:
Yeah, I plan to rewrite that part in Poetry to generate the error message without the use of crashtest. I discussed it with the team and since Poetry uses crashtest for only one error, it will be easy to remove it.

Reply:
Sounds good to me